pkg/kube/pod.go

```diff
@@ -105,6 +107,15 @@ func CreatePod(ctx context.Context, cli kubernetes.Interface, opts *PodOptions)
 		},
 		Spec: patchedSpecs,
 	}
+
+	// Add Annotations and Labels, if specified
+	if opts.Annotations != nil {
+		pod.ObjectMeta.Annotations = opts.Annotations
+	}
+	if opts.Labels != nil {
+		pod.ObjectMeta.Labels = opts.Labels
+	}
```

Comment:
Is there any form of validation required for the labels and annotations ? Or is it fine to copy everything ?

Reply:
Not really required since they are just a `map[string]string` and could have anything as key/value pairs